tox.ini

```diff
@@ -5,6 +5,9 @@ envlist =
     black
     flake8
     docs
+    pdf
+    2fa
+    axes
```

Comment:
I'm wondering if we can make that additional variants of the main envlist?

```
py{310,311,312}-django{42,52}-{plain,pdf,mfa,axes}
py{313}-django{52}-{plain,pdf,mfa,axes}
```
etc.

Reply:
got it working!